AUTHORS

```diff
@@ -5,6 +5,18 @@ community. It is currently maintained by the following people:
    Rene Gassmoeller
    Timo Heister
 
+The following people have significantly contributed and furthered ASPECT's
+goals and are therefore official project developers:
```

Comment:
Maybe use upper-case Project Developers to make clear that that's a title? Or, project Developers? (I forgot whether we settled on a formal title.)